src/components/AssetInformation.jsx

```diff
@@ -21,6 +22,24 @@ const AssetInformation = ({
     </div>
 );
 
+AssetInformation.propTypes = {
+    price       : PropTypes.string,
```

Comment:
there are lots of extra space here and in `AssetInformation.defaultProps `, is that needed?